index.html

```diff
@@ -223,6 +223,7 @@ <h1 style="color: #ffffff; text-align: center;">MBDyn Logo Designs</h1>
             <section class="latest-issues">
                 <h2>LATEST ISSUES</h2>
                 <div class="issues-wrapper">
+                  <div class="button"><a href=''>See all issues on&nbsp;<i class="fa fa-github"></i>github</button>
```

Comment:
Line is longer than allowed. (115 > 100)

*LineLengthBear, severity NORMAL, section `default`.*